tfhe/cbindgen.toml

```diff
@@ -114,14 +114,15 @@ bitflags = false
 ############## Options for How Your Rust library Should Be Parsed ##############
 
 [parse]
-parse_deps = true
-include = ["tfhe"]
+parse_deps = false
+include = []
```

Comment:
What does this change? Why is this necessary?

Reply:
it was a mistake from the time the C API required macro expansion, I think I made the mistake at the time, we don't need to parse deps, only our own crate, the include here is not required as it refers to dependencies as far as I understand the cbindgen doc, the parse expand is handled in the build.rs to programatically feed features that are enabled